src/hello-world.md

```diff
@@ -70,20 +48,18 @@ Alternatively, if you are running locally without any webserver.
 </html>
 ```
 
-If you have Python 3 installed, you can alternatively serve this file with Python's built 
-in web server from `localhost`. Python's web server cannot serve `instantiateStreaming` or 
-`compileStreaming` due to its lack of support for the `application/wasm` MIME type.
+Then, (assuming you have `python3`) installed, run
```

Comment:
You've already made the assumption before that someone has python3 above though. I'm with @fitzgen in that we should keep the instantiate streaming things